python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -57,16 +58,19 @@ def evaluate(
         group_metrics_min_queries: int = 50,
         logs_dir: Optional[str] = None,
         logging_frequency: int = 25,
+        track_experiment: bool = False,
     ):
         """
         Evaluate the ranking model
 
         Args:
             test_dataset: an instance of tf.data.dataset
             inference_signature: If using a SavedModel for prediction, specify the inference signature
+            additional_features: Additional post processing feature functions as key value pairs
```

Comment:
@lastmansleeping It's a little confusing to me that some of these args are accessible through `parse_args` and some are not. Is that intentional/what's the reasoning?

Reply:
Yeah, this is intentional. These arguments are for using ml4ir as a library with custom functions. So you'll be plugging them in here as a dictionary of functions. More documentation coming soon.

Reply:
Ok. Good enough for me.